docs/book/user-guide/tutorials/trigger-pipelines-from-external-systems.md

```diff
@@ -0,0 +1,590 @@
+    """Train a model based on the specified type."""
+    X_train = datasets['X_train']
+    y_train = datasets['y_train']
+    
+    if model_type == "random_forest":
+        model = RandomForestClassifier(n_estimators=100, random_state=42)
+    elif model_type == "gradient_boosting":
+        model = GradientBoostingClassifier(random_state=42)
+    else:
+        raise ValueError(f"Unknown model type: {model_type}")
+    
+    print(f"Training a {model_type} model...")
+    model.fit(X_train, y_train)
+    return model
+
+@step
+def evaluate(
+    datasets: Dict[str, Any], 
+    model: Union[RandomForestClassifier, GradientBoostingClassifier]
+) -> Dict[str, float]:
+    """Evaluate the model and return metrics."""
+    X_test = datasets['X_test']
+    y_test = datasets['y_test']
+    
+    y_pred = model.predict(X_test)
+    accuracy = accuracy_score(y_test, y_pred)
+    
+    print(f"Model accuracy: {accuracy:.4f}")
+    return {'accuracy': float(accuracy)}
+
+
+@pipeline
+def training_pipeline(
+    data_url: str = "s3://example-bucket/data.csv",
+    model_type: str = "random_forest"
+):
+    """A configurable training pipeline that can be triggered externally."""
+    data = load_data(data_url)
+    datasets = preprocess(data)
+    model = train_model(datasets, model_type)
+    metrics = evaluate(datasets, model)
+
+# For local execution during development
+if __name__ == "__main__":
+    # Run with default parameters
+    training_pipeline()
+```
+
+This pipeline is designed to be configurable with parameters that might change between runs:
+- `data_url`: Where to find the input data
+- `model_type`: Which algorithm to use
+
+These parameters make it an ideal candidate for external triggering scenarios where we want to run the same pipeline with different configurations.
+
+## Method 1: Using Run Templates (ZenML Pro)
+
+{% hint style="success" %}
+This is a [ZenML Pro](https://zenml.io/pro)-only feature. Please [sign up here](https://cloud.zenml.io) to get access.
+{% endhint %}
+
+[Run Templates](https://docs.zenml.io/how-to/trigger-pipelines) are the most straightforward way to trigger pipelines externally in ZenML. They provide a pre-defined, parameterized configuration that can be executed via multiple interfaces.
+
+### Creating a Run Template
+
+First, we need to create a template based on our pipeline. This requires having a remote stack with at least a remote orchestrator, artifact store, and container registry.
+
+#### Using Python:
+
+```python
+from zenml.client import Client
+
+# First get the pipeline by name
+pipeline = Client().get_pipeline("training_pipeline")
+
+# Get the most recent runs for this pipeline
+runs = Client().list_pipeline_runs(
+    pipeline_id=pipeline.id, 
+    sort_by="desc:created", 
+    size=1
+)
+
+if runs:
+    # Use the most recent run
+    latest_run = runs[0]
+    
+    # Create a template from this run
+    template = Client().create_run_template(
```

Comment:
mention you can pass a default config to this that sets up the default parameters